docs/requirements-docs.txt

```diff
@@ -3,3 +3,4 @@ sphinx-docstring-typing
 urllib3
 requests
 requests-oauthlib
+pyjwt
```

Comment:
what happens if you do not add this dependency here?

Reply:
I think we need this dependency for documentation, so removing it also works, removed the dependency and updated the pull request